activerecord/test/cases/adapters/postgresql/range_test.rb

```diff
@@ -163,6 +168,12 @@ def test_custom_range_values
     assert_nil @empty_range.float_range
   end
 
+  def test_custom_range_value_escaping
+    insert_range(id: 106, string_range: '["ca""t","do\\g")')
+    escaped_range = PostgresqlRange.find(106)
+    assert_equal("ca\\t"..."do\"g", escaped_range.string_range)
```

Comment:
It is weird why the assertion is passed. Looks like `insert_range` doesn't support `string_range`.

https://github.com/rails/rails/blob/b119b4c4bf86d34fd6063fdbe4796dec30781a82/activerecord/test/cases/adapters/postgresql/range_test.rb#L433-L455

Reply:
I'll fix that, but that _is_ weird!